### PR TITLE
AliECS dump: allow to enable/disable logging stdout and stderr separately

### PR DESCRIPTION
**Pull request by knopers8**

### CODE REVIEW

File: Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx [hunk 1]
```diff
@@ -133,7 +133,8 @@ const auto expectedWorkflow = R"EXPECTED(name: testwf
 const std::vector expectedTasks{
   R"EXPECTED(name: A
 defaults:
-  log_task_output: none
+  log_task_stdout: false
+  log_task_stderr: false
```
Comment (teo) [inline Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx]: Should be `none`, not `false`

File: Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx [hunk 2]
```diff
@@ -220,7 +222,8 @@ const std::vector expectedTasks{
 )EXPECTED",
   R"EXPECTED(name: B
 defaults:
-  log_task_output: none
+  log_task_stdout: false
+  log_task_stderr: false
```
Comment (teo) [inline Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx]: `none`

File: Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx [hunk 3]
```diff
@@ -307,7 +311,8 @@ const std::vector expectedTasks{
 )EXPECTED",
   R"EXPECTED(name: C
 defaults:
-  log_task_output: none
+  log_task_stdout: false
+  log_task_stderr: false
```
Comment (teo) [inline Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx]: `none`

File: Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx [hunk 4]
```diff
@@ -394,7 +400,8 @@ const std::vector expectedTasks{
 )EXPECTED",
   R"EXPECTED(name: D
 defaults:
-  log_task_output: none
+  log_task_stdout: false
+  log_task_stderr: false
```
Comment (teo) [inline Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx]: `none`

**Review by teo — CHANGES_REQUESTED** [review]

File: Framework/Core/src/O2ControlHelpers.cxx [hunk 5]
```diff
@@ -367,7 +368,8 @@ void dumpTask(std::ostream& dumpOut, const DeviceSpec& spec, const DeviceExecuti
 {
   dumpOut << indLevel << "name: " << taskName << "\n";
   dumpOut << indLevel << "defaults:\n";
-  dumpOut << indLevel << indScheme << "log_task_output: none\n";
+  dumpOut << indLevel << indScheme << "log_task_stdout: false\n";
+  dumpOut << indLevel << indScheme << "log_task_stderr: false\n";
```
Comment (teo) [inline Framework/Core/src/O2ControlHelpers.cxx]: `none`

File: Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx (hunk 1 above)
Comment (knopers8) [reply Framework/Core/test/test_FrameworkDataFlowToO2Control.cxx]: riiight

**Review by teo — APPROVED** [review]

**Comment by knopers8** [comment]
Could someone review && merge please?

**Comment by vascobarroso** [comment]
@AliceO2Group/framework-admins can we get this change in one of the next O2 tags ? We would like to deploy it next Monday at P2.

**Pull request merged**